synchronizer/synchronizer.go

```diff
@@ -705,6 +721,10 @@ func (s *ClientSynchronizer) processGlobalExitRoot(globalExitRoot etherman.Globa
 		log.Fatalf("networkID: %d, error storing the GlobalExitRoot in processGlobalExitRoot. BlockNumber: %d, error: %s",
 			s.networkID, globalExitRoot.BlockNumber, err.Error())
 	}
+	if s.l1RollupExitRoot != globalExitRoot.ExitRoots[1] {
+		s.l1RollupExitRoot = globalExitRoot.ExitRoots[1]
+		s.chExitRootEvent <- &globalExitRoot
```

Comment:
I don't fully understand why we monitor the rollupexitroot instead the mainnetexitroot. The second one will allow us to do claims in l2 and the other one will only allow users to do claims in l1

Reply:
yes, we will update `ready_for_claim` field directly when getting `exit_root` updates. It allows us to reduce query in bridge rest api to check the claimable status